src/client/comments/commentsActions.js

```diff
@@ -102,24 +143,31 @@ export const sendComment = (parentPost, body, isUpdating = false, originalCommen
   return dispatch({
     type: SEND_COMMENT,
     payload: {
-      promise: steemConnectAPI
-        .comment(parentAuthor, parentPermlink, author, permlink, '', newBody, jsonMetadata)
-        .then((resp) => {
-          const focusedComment = {
-            author: resp.result.operations[0][1].author,
-            permlink: resp.result.operations[0][1].permlink,
-          };
-          dispatch(notify('Comment submitted successfully', 'success'));
-          dispatch(getComments(id, true, focusedComment));
-
-          if (window.analytics) {
-            window.analytics.track('Comment', {
-              category: 'comment',
-              label: 'submit',
-              value: 3,
-            });
-          }
-        }),
+      promise: broadcastComment(
+        steemConnectAPI,
+        parentAuthor,
+        parentPermlink,
+        author,
+        permlink,
+        '',
+        newBody,
+        jsonMetadata,
+      ).then((resp) => {
+        const focusedComment = {
+          author: resp.result.operations[0][1].author,
```

Comment:
I wish there was a better way to retrieve the author rather than this line `resp.result.operations[0][1]`, looks like a lot of things can go wrong here since you assume that `result.operations` is an array and the first index is the one you want get then you get the 2nd index of that lol... oh well i guess thats what the response returns, can't be helped i guess.